### PR TITLE
OSS-Fuzz: OSS-Fuzz integration and new fuzzers

### PR DESCRIPTION
**Pull request by arthurscchan**
Hi! Would you be interested in setting up fuzzing via OSS-Fuzz?

I noticed that this project already has a Cargo Fuzz setup with existing fuzzers. This PR further expands the fuzzing coverage by adding a second fuzzer to the configuration. I have also created an initial integration in https://github.com/google/oss-fuzz/pull/12628. If you're happy to proceed with the integration, please let me know, and I'd be happy to elaborate on any points if needed.

The only thing required at this point for integration is an email associated with a Google account, which will be used to receive notifications when bugs are found.

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
Just noticed https://github.com/pyfisch/httpdate/issues/17 -- apologies for the double issue/PR @LeoDog896

File: fuzz/Cargo.toml [hunk 1]
```diff
@@ -20,3 +20,7 @@ members = ["."]
 [[bin]]
 name = "fuzz_target_1"
 path = "fuzz_targets/fuzz_target_1.rs"
```
Comment (pyfisch) [inline fuzz/Cargo.toml]: Maybe also rename `fuzz_target_1` to something more descriptive?

File: fuzz/fuzz_targets/fuzz_conversion.rs [hunk 2]
```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 Google LLC
```
Comment (pyfisch) [inline fuzz/fuzz_targets/fuzz_conversion.rs]: I'm a bit confused that the file contains a copyright header (other files in the crate don't) and that it's Copyright Google LLC while you @arthurscchan do not appear to work for Google?

**Review by pyfisch — CHANGES_REQUESTED** [review]
Thank you for adding an additional fuzzer! I don't think there is a need for continuous fuzzing by Google as this is a really small crate  but if one issue is found that is great.

File: fuzz/Cargo.toml (hunk 1 above)
Comment (arthurscchan) [reply fuzz/Cargo.toml]: I have renamed it. Thanks for your comments.

File: fuzz/fuzz_targets/fuzz_conversion.rs (hunk 2 above)
Comment (arthurscchan) [reply fuzz/fuzz_targets/fuzz_conversion.rs]: I have remove the confusing license.

**Comment by arthurscchan** [comment]
@pyfisch I have fixed the name of the fuzzing harness and remove the misleading license. Is it OK to add your email of your github account to OSS-Fuzz thus you could receive notification when OSS-Fuzz does discover some issues?

**Comment by pyfisch** [comment]
@arthurscchan Thank you for adding the fuzzer! You can add my email address to OSS-Fuzz.

**Pull request merged**